contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,332 @@
+
+## Indicator #1: Organic word-of-mouth growth
+
+To be clear... you _can't_ validate product-market fit _using word-of-mouth alone_. That way lies madness. But it is a useful leading indicator _when confirmed using other metrics_, such as user engagement, and retention.
+
+Hopefully, you already have a strong grasp of your organic user growth via user signup, or whatever metric makes sense for your product.
+
+But you can, with a little work, also track word of mouth awareness and sentiment of your product. Here are a few options.
+
+### 1. Brand mention alerts
+
+Use a tool like [Syften](https://syften.com/) to monitor social media, communities, newsletters, etc., for mentions of any keyword you choose, such as your brand name. We use Syften at PostHog to send alerts to a `#brand-alerts` channel on our Slack. 
+
+You can go one step further by setting up a Zapier integration with Google Sheets, where you can collect your mentions, and track trends over time.
+
+Brand alerts are also a great way to drive acquisition by talking to people considering your product. 
+
+> **❗️Important:** It's _much easier_ to track mentions if your company has a unique name. You're bang out of luck if it's a common noun – e.g. Apple, Amplitude. 
+
+### 2. Searches for your brand
+
+**Google Trends** is a good option if you have a unique brand name. Just plug your brand into Trends to see how searches for your brand are trending over time.
+
+![google trend for posthog](../images/blog/how-to-measure-product-market-fit/posthog-trends.png)
+
+It's also useful for comparing yourself to other companies, such as those you _know_ have product-market fit, or competitors.
+
+![google trend comparison](../images/blog/how-to-measure-product-market-fit/posthog-with-logrocket.png)
+
+At PostHog, we also use [Glimpse](https://meetglimpse.com/) – a handy extension that augments Trends by adding trend lines, forecasts, and real search volumes. 
+
+![trends with Glimpse](../images/blog/how-to-measure-product-market-fit/glimpse.png)
+
+It'll also suggest similar companies you can compare yourself against.
+
+![related companies](../images/blog/how-to-measure-product-market-fit/related-companies.png)
+
+**Google Search Console** is an alternative if you don't have a unique brand name, but the data isn't 100% reliable, so you're better off using the next method.
+
+### 3. Organic traffic to your homepage
+
+One final way to track word-of-mouth is organic users who visit your website homepage as a proxy for word-of-mouth. We track this in PostHog as an insight that combines:
+
+- Unique users who visited the homepage via Google search
+- Unique users how visited the homepage directly
+
+This captures people who search for your brand, and those who navigate there directly, or visit via Slack, WhatsApp, etc. 
+
+This method will capture some existing users as well, but we've found it correlates well with surges in word-of-mouth awareness.
```

Comment:
```suggestion
This method will capture some existing users as well, but we've found it correlates well with surges in word-of-mouth awareness.

![posthog proxy](../images/blog/how-to-measure-product-market-fit/word-of-mouth-proxy.png)

Above you can see the trend for our homepage since the beginning of 2021. Unsurprisingly, it's similar to that of  Google Trends. 
```